index.html

```diff
@@ -672,12 +714,13 @@ <h2>Privacy Considerations</h2>
 
     <section>
         <h2>Grafting Peer DIDs Into Another DID Method Namespace</h2>
-        <p>Because peer DIDs are guaranteed to be globally unique at the moment of creation,
-            their 128-bit NSI will not exist on any other blockchain. Blockchain-based DID methods
+        <p>Because peer DIDs are globally unique at the moment of creation,
+            their <em>numeric basis</em> will not exist on any other blockchain unless until someone
```

Comment:
"unless until" - I think you mean "until" ?

Reply:
Fixed.